code-the-dream-react/src/TodoList.js

```diff
@@ -17,7 +18,11 @@ const todoList = [
 
 function TodoList() {
     return (
-      
+      <ul>
+        {todoList.map((todoListChunk) => (
+          <TodoListItem key={todoListChunk.objectID} propsTodoListChunk={todoListChunk} />
```

Comment:
- The name for the prop can be simplified such  as `listItem` 
-  We want to avoid naming react `props` with the word `prop` the way you did it for `propsTodoListChunk`, a better name, in my opinion, could just have been `listItem`.

Line 23 could be summarized like so:
```js
<TodoListItem key={todoListChunk.objectID} listItem={listItem} />
```
it is usually a good convention to list your prop with the same name as your value.

Let me know if you have any other questions.

Reply:
I changed the name of my prop. Thank you for letting me know!